packages/server/lib/hooks/connection/verification/lastpass-verification.ts

```diff
@@ -0,0 +1,41 @@
+import type { InternalNango as Nango } from '../verificatiion-script.js';
+import type { UserResponse } from '../../response-types/lastpass.js';
+
+export default async function execute(nango: Nango) {
+    const { credentials, providerConfigKey } = await nango.getCredentials();
+
+    if (!isValidCredentials(credentials)) {
+        throw new Error('Invalid credentials format');
+    }
```

Comment:
I'm not entirely sure this can happen, when you call this method it's already been through zod and credentials formatting